### DISCUSSION TITLE
Database 구조

### DISCUSSION THREAD

**Discussion by mypark42** [GunGonGamLeeLee/ft_transcendence#26] [3. Problem & Solving]
<img width="809" alt="Screen Shot 2022-10-05 at 1 37 27 PM" src="https://user-images.githubusercontent.com/56795942/193982453-55fbea05-f6d8-4c03-96ea-44cfd1538b65.png">

**Comment by mypark42** [comment]
? : boolean 타입

**Comment by mypark42** [comment]
https://www.erdcloud.com/d/SRzLxXmy5La9FBds7

**Reply by mypark42** [reply]
처음에 구조 잘못 생각해서 고침..

**Comment by niamu01** [comment]
![transcendence](https://user-images.githubusercontent.com/74657997/194807749-4e268262-d255-4e83-91e3-39be9f777d1c.png)
